scripts/close-milestone.sh

```diff
@@ -0,0 +1,11 @@
+set -x -e -o pipefail -u
+
+indented_milestones_json_array=$(gh api -H "Accept: application/vnd.github+json" /repos/VirtusLab/git-machete-intellij-plugin/milestones)
+
+desired_milestone=$( echo $indented_milestones_json_array | jq -c --arg TITLE $1 '.[] | select(.title | contains($TITLE))')
+
+number=$(echo $desired_milestone | jq '.number')
```

Comment:
Nit: just append `| .number` to the jq expression in the preceding line, no need for a separate statement (as `desired_milestone` is otherwise unused)